dashboard/app/views/plc/user_course_enrollments/index.html.haml

```diff
@@ -9,24 +9,27 @@
   - @user_course_enrollments.each do |user_course_enrollment|
     %h2 #{user_course_enrollment.plc_course.name}
     - user_course_enrollment.plc_unit_assignments.each do |enrollment_unit_assignment|
+      - next if enrollment_unit_assignment.status == Plc::EnrollmentUnitAssignment::START_BLOCKED
+
+      - content_module_assignment = enrollment_unit_assignment.module_assignment_for_type Plc::LearningModule::CONTENT_MODULE
+      - practice_module_assignment = enrollment_unit_assignment.module_assignment_for_type Plc::LearningModule::PRACTICE_MODULE
+
       .outlined_section
         %h4
           = enrollment_unit_assignment.plc_course_unit.unit_name
-        - case enrollment_unit_assignment.status
-          - when Plc::EnrollmentUnitAssignment::START_BLOCKED
-            %p
-              This unit cannot be started yet
-          - when Plc::EnrollmentUnitAssignment::PENDING_EVALUATION
-            %p
-              = link_to 'Click here to take the evaluation quiz for this unit', perform_evaluation_path(enrollment_unit_assignment), class: 'btn'
-          - when Plc::EnrollmentUnitAssignment::IN_PROGRESS
-            %p
-              = link_to 'Click here to view tasks, resources, and get badged', plc_enrollment_unit_assignment_path(enrollment_unit_assignment), class: 'btn'
-          - when Plc::EnrollmentUnitAssignment::COMPLETED
-            %p.fa.fa-check-circle.fa-5x{style: 'color: green; display: table; margin: auto'}
-            %p{style: 'text-align: center'}
-              #{enrollment_unit_assignment.plc_course_unit.unit_name} badge complete!
-            %p
-              = link_to "Click here to #{enrollment_unit_assignment.plc_course_unit.unit_name} resources", plc_enrollment_unit_assignment_path(enrollment_unit_assignment), class: 'btn'
+        .course_unit_sections
```

Comment:
Every unit is guaranteed to have a practice and content section, as well as peer reviews. All units will have an assessment section and possibly a unit overview section. 

There's no plans for unit extras, and if that's in scope now then I'm going to have to change a lot more than just this view. I'll confirm with Brook if this is feasible.

I don't follow what the LearningModule description has to do with this section or icon - Unit Content will take you to many modules, one of which is your assigned focus area. Are you saying that the description of that focus area should be displayed here?